resources/js/Components/Tables/RoleTable.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <div class="table-container">
+    <table class="table-auto responsive-spaced">
+      <caption></caption>
+      <thead>
+        <tr>
+          <th class="lt-grey p-3" id="id_role_name">Role Name</th>
+          <th class="lt-grey p-3" id="id_role_set">Permission Set</th>
+          <th class="lt-grey p-3" id="id_role_action">Action</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="role in roles" :key="role.id">
+          <td class="text-center lt-grey p-3">{{role.name}}</td>
+          <td class="text-center lt-grey p-3">
+            <Dropdown width="48">
```

Comment:
Do we want to match the way call other vue components in this file (kebab-case - ex: `<update-role-form>`)

Reply:
Yes will change this

Reply:
Closing